app/views/RoomActionsView/ActionsList/index.js

```diff
@@ -0,0 +1,222 @@
+import React, { memo } from 'react';
+import { View, ScrollView } from 'react-native';
+import { isEqual } from 'lodash';
+import PropTypes from 'prop-types';
+
+import I18n from '../../../i18n';
+import RocketChat from '../../../lib/rocketchat';
+
+import ActionItem from './ActionItem';
+import RoomItem from './RoomItem';
+
+import scrollPersistTaps from '../../../utils/scrollPersistTaps';
+import styles from './styles';
+
+const areEqual = (prev, next) => isEqual(prev.room, next.room)
+	&& prev.baseUrl === next.baseUrl
+	&& prev.member._id === next.member._id
+	&& prev.membersCount === next.membersCount
+	&& prev.canViewMembers === next.canViewMembers
+	&& prev.canAddUser === next.canAddUser
+	&& prev.canAutoTranslate === next.canAutoTranslate
+	&& prev.joined === next.joined
+	&& prev.jitsiEnabled === next.jitsiEnabled;
+
+const ActionsList = memo(({
+	room,
+	baseUrl,
+	user,
+	member,
+	membersCount,
+	canViewMembers,
+	canAddUser,
+	canAutoTranslate,
+	joined,
+	jitsiEnabled,
+	handleShare,
+	toggleBlockUser,
+	leaveChannel,
+	navigation
+}) => {
+	const { navigate } = navigation;
+	const {
+		rid, t, blocker
+	} = room;
+
+	return (
+		<ScrollView {...scrollPersistTaps} contentContainerStyle={styles.container}>
+
+			<RoomItem
+				name={I18n.t('Room_Info')}
+				baseUrl={baseUrl}
+				route='RoomInfoView'
+				params={{ rid, t, room }} // forward room only if room isn't joined
+				user={user}
+				member={member}
+				testID='room-actions-info'
+			/>
+			<View style={[styles.sectionSeparator, styles.sectionSeparatorBorder]} />
+
+			{jitsiEnabled && (
+			<>
+				<ActionItem
+					icon='livechat'
+					name={I18n.t('Voice_call')}
+					onPress={() => RocketChat.callJitsi(rid, true)}
+					testID='room-actions-voice'
+				/>
+				<View style={styles.separator} />
+				<ActionItem
+					icon='video'
+					name={I18n.t('Video_call')}
+					onPress={() => RocketChat.callJitsi(rid)}
+					testID='room-actions-video'
+				/>
+				<View style={[styles.sectionSeparator, styles.sectionSeparatorBorder]} />
+			</>
+			)}
+
+			{(t === 'c' || t === 'p') && canViewMembers && (

```

Comment:
`(t === 'c' || t === 'p')` repeat a lot of times, what you think about extract it to a variable?